Explorer/Assets/DCL/PerformanceAndDiagnostics/Diagnostics/ReportsHandling/ReportHub.cs

```diff
@@ -15,6 +14,7 @@ public static class ReportHub
             {
                 (ReportHandler.DebugLog, new DefaultReportLogger()),
             });
+        public static bool LogVerboseEnabled = false;
```

Comment:
I suggest to make it private and create Initialize method with assigning logger field as well

Reply:
sounds good to me, implemented at https://github.com/decentraland/unity-explorer/pull/1546/commits/0d20de12ee24b27c5ff92c7bcffe190a680f4cd7